jenkins/envs/daint_clang_nvcc.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+source $(dirname "$BASH_SOURCE")/daint.sh
+
+module load PrgEnv-gnu
+module swap gcc/7.3.0
```

Comment:
can you add a comment why you load gcc? I guess because of gfortran?

Reply:
It’s actually not strictly needed when I remember correctly, we could also use the default gfortran. But further down, I also tell Clang to use the gcc 7.3.0 toolchain, so it probably makes sense for consistency…